provision/travis/run-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Run tests on Travis CI worker.
+
+#Exit on error
+set -e
+#enable logging
+set -x
+
+if [ -z "$TEST_ENV" ]; then
+#No test environment is set, we run all the tests, build the docs and do black
+    pytest --cov=tonic/ tonic test
+    sphinx-build -b html doc/source/ doc/build/
+    black --check tonic/
+elif [ "$TEST_ENV" = "py36-test" ]; then
+    pytest --cov=tonic/ tonic test
+elif [ "$TEST_ENV" = "doc" ]; then
+    sphinx-build -b html doc/source/ doc/build/
+elif [ "$TEST_ENV" = "black" ]; then
+     black --check tonic/
```

Comment:
some repeated lines of code, is it possible to rewrite this so the sphinx, pytest and black lines are only written once?

Reply:
also do black on whole project, including tests.

Reply:
Test invocations are declared at beginning of the bash file now.
black runs also checks the test/ directory